ivy_tests/test_docstrings.py

```diff
@@ -222,75 +347,25 @@ def test_docstrings(backend):
     ivy.set_backend(backend)
     failures = list()
     success = True
-    """
-    Functions skipped as their output dependent on outside factors:
-
-    random_normal, random_uniform, shuffle, num_gpus, current_backend,
-    get_backend
-    """
-    to_skip = [
-        "random_normal",
-        "random_uniform",
-        "randint",
-        "shuffle",
-        "beta",
-        "gamma",
-        "dev",
-        "num_gpus",
-        "current_backend",
-        "get_backend",
-        "namedtuple",
-        "invalid_dtype",
-        "DType",
-        "NativeDtype",
-        "Dtype",
-        "multinomial",
-        "num_cpu_cores",
-        "get_all_ivy_arrays_on_dev",
-        "num_ivy_arrays_on_dev",
-        "total_mem_on_dev",
-        "used_mem_on_dev",
-        "percent_used_mem_on_dev",
-        "function_supported_dtypes",
-        "function_unsupported_dtypes",
-        "randint",
-        "unique_counts",
-        "unique_all",
-        "total_mem_on_dev",
-        "supports_inplace_updates",
-        "get",
-        "deserialize",
-        "dropout",
-        "dropout1d",
-        "dropout3d",
-    ]
-    # the temp skip list consists of functions which have an issue with their
-    # implementation
-    skip_list_temp = [
-        "outer",
-        "argmax",
-        "split",
-        "det",
-        "cumprod",
-        "where",
-        "sinc",
-        "grad",
-    ]
 
     # skip list for array and container docstrings
     skip_arr_cont = [
+        # generates different results due to randomization
```

Comment:
It would be great if this list was also defined in the same place as the other 2 lists, just to make it easier to find